client/dashboard/app/queries/site-jetpack-settings.ts

```diff
@@ -0,0 +1,23 @@
+import { fetchJetpackSettings, updateJetpackSettings } from '../../data/site-jetpack-settings';
+import { queryClient } from '../query-client';
+import { siteQueryFilter } from './site';
+import type { SiteSettings } from '../../data/site-settings';
+
+export const siteJetpackSettingsQuery = ( siteId: number ) => ( {
```

Comment:
Please wrap the query and mutation objects 😄 see the recently merged PR:

- https://github.com/Automattic/wp-calypso/pull/104975